docs/tutorials/qaoa.ipynb

```diff
@@ -623,12 +493,12 @@
     "id": "auo2VuTm6haO"
    },
    "source": [
-    "The QAOA is known to do just a little better better than random guessing for Max-Cut on 3-regular graphs at `p=1`. You can use very similar logic to the code above, but now instead of relying on the QAOA to decied your `S_partition` and `T_partition` you can just pick then randomly:"
+    "The QAOA is known to do just a little better than random guessing for Max-Cut on 3-regular graphs at `p=1`. You can use very similar logic to the code above, but now instead of relying on the QAOA to decide your `S_partition` and `T_partition` you can just pick then randomly:"
```

Comment:
> nittiest of nit nits: extra space after better when you removed the extra better

@dabacon It looks like a reviewnb bug -- there's no extra space in text as seen here. Merging now.